docs/plugins/preset-stage-2.md

```diff
@@ -9,6 +9,7 @@ package: babel-preset-stage-2
 This preset includes the following plugins:
 
 - [transform-object-rest-spread](/docs/plugins/transform-object-rest-spread)
+- [transform-decorators](/docs/plugins/transform-decorators)
```

Comment:
I think we can keep the message from before since the transform still isn't updated

I added https://github.com/babel/babel/pull/3626/files as a warning for the default one